### DISCUSSION TITLE
Does `a2wsgi` plan to support websocket conversion?

### DISCUSSION THREAD

**Discussion by Archmonger** [abersheeran/a2wsgi#36] [Q&A] [answered]
Does `a2wsgi` plan to support ASGI to WSGI websocket conversion?

Currently, websockets are supported on `werkzeug`, `gunicorn`, `eventlet`, and `gevent` [in a format that isn't defined](https://github.com/benoitc/gunicorn/blob/master/examples/websocket/websocket.py) in the WSGI spec.

We are currently looking into whether we can rely on `a2wsgi` in the future to simplify our WSGI compatibility over at [ReactPy](https://github.com/reactive-python/reactpy).

A practical example of a WSGI WebSocket can be found over at [flask-sock](https://github.com/miguelgrinberg/flask-sock/blob/main/src/flask_sock/__init__.py).

**Comment by abersheeran** [comment, marked as answer]
This sounds interesting and I'd love to try it. If anyone wants to make a PR, I can help too.
